### PR TITLE
Improve type check for driver none

### PR DESCRIPTION
**Pull request by medyagh**

### CODE REVIEW

File: pkg/minikube/cluster/cluster.go [hunk 1]
```diff
@@ -72,7 +72,7 @@ func init() {
 
 // CacheISO downloads and caches ISO.
 func CacheISO(config cfg.MachineConfig) error {
-	if config.VMDriver != "none" {
+	if config.VMDriver == constants.DriverNone {
```
Comment (afbjorklund) [inline pkg/minikube/cluster/cluster.go]: This logic looks like it has been reversed ?

Comment (medyagh) [reply pkg/minikube/cluster/cluster.go]: ooops good catch. thanks for catching it

**Comment by sharifelgamal** [comment]
@minikube-bot OK to test

File: pkg/minikube/cluster/cluster.go (hunk 1 above)
Comment (medyagh) [reply pkg/minikube/cluster/cluster.go]: Done

**Pull request merged**